bpf/lib/process.h

```diff
@@ -155,6 +155,18 @@ struct msg_execve_key {
 	__u64 ktime;
 }; // All fields aligned so no 'packed' attribute.
 
+/* Execution and cred related flags shared with userspace */
+#define EXEC_SETUID 0x01
+#define EXEC_SETGID 0x02
+
+/* This is the struct stored in bpf map to share info between
+ * different execve hooks.
+ */
+struct execve_info {
+	__u32 secureexec;
+	__u32 pad;
+};
```

Comment:
Why the padding here? Can't we just have a __u32 type value?
The name `secureexec` seems confusing to me. How about just `flags`?

Reply:
for the padding IIRC entries are aligned on 8bytes...

For the securexec let me see, in the kernel there is bprm->secureexec bit field to indicate this, I can do flag or better document along with how kernel do things

Reply:
Ok I left it secureexec but extensive documentation where it comes from, flags is too generic for this.